CppWinRTProjectionSample/SimpleMathProjection/SimpleMathProjection.csproj

```diff
@@ -0,0 +1,80 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0-windows10.0.19041.0</TargetFramework>
+    <Platforms>x64</Platforms>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <GenerateTestProjection>true</GenerateTestProjection>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <!--<NoWarn>8305;0618</NoWarn>-->
+    <GeneratedFilesDir>.\Generated Files\</GeneratedFilesDir>
+  </PropertyGroup>
+  
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Windows.CsWinRT" Version="0.1.0-prerelease.200922.1" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\SimpleMathComponent\SimpleMathComponent.vcxproj" />
+  </ItemGroup>
+
+  <!--C#/WinRT Targets and Props-->
+  
+  <!--Identify WinMDs in project references-->
+  <Target Name="RemoveWinMDReferences" BeforeTargets="ResolveReferences" AfterTargets="AfterResolveReferences">
+    <ItemGroup>
+      <!--Track WinMDs-->
+      <ReferenceWinMDs Include="@(ReferencePath)" Condition="'%(ReferencePath.WinMDFile)' == 'true'" />
+      <!--Remove WinMDs from project outputs-->
+      <ReferencePath Remove="@(ReferencePath)" Condition="'%(ReferencePath.WinMDFile)' == 'true'" />
+    </ItemGroup>
+  </Target>
+
+  <Target Name="GenerateProjection" Condition="'$(GenerateTestProjection)' == 'true'">
+
+    <!--Project Property for which namespaces to include-->
+    <PropertyGroup>
+      <ProjectionNamespaceToInclude>SimpleMathComponent</ProjectionNamespaceToInclude>
+    </PropertyGroup>
+
+    <!--CSWinRT properties-->
+    <PropertyGroup>
+      <CsWinRTVerbosity>high</CsWinRTVerbosity>
+      <CsWinRTResponseFile>$(GeneratedFilesDir)cswinrt_build_projection.rsp</CsWinRTResponseFile>
+      <CsWinRTCommand>$(CsWinRTExe) %40"$(CsWinRTResponseFile)"</CsWinRTCommand>
+        <!---target $(TargetFramework)-->
+    <CsWinRTParams>
+        -verbose
+        -in 10.0.18362.0
```

Comment:
$(WindowsSDKVersion.TrimEnd('\')) could be captured in directory.build.props and used throughout (again, could do this in a separate issue)